src/mono/sample/iOS/Program.csproj

```diff
@@ -28,6 +29,9 @@
 
   <ItemGroup>
     <RuntimeComponents Condition="'$(DiagnosticPorts)' != ''" Include="diagnostics_tracing" />
+    <EnvironmentVariables Condition="'$(UseMonoRuntime)' != 'true'" Include="DOTNET_InterpMode=3" />
+    <EnvironmentVariables Condition="'$(UseMonoRuntime)' != 'true'" Include="DOTNET_ReadyToRun=0" />
+    <EnvironmentVariables Condition="'$(UseMonoRuntime)' != 'true'" Include="DOTNET_InterpDump=*!*" />
```

Comment:
Just for my knowledge, what does this environment variable do?

Reply:
DOTNET_InterpMode - Enables Interpreter for a subset of all methods depending on the specified mode
DOTNET_ReadyToRun - Enable/disable use of ReadyToRun native code
DOTNET_InterpDump - Dumps all compiled methods